data_sharing.md

```diff
@@ -0,0 +1,54 @@
+# Data Sharing
+
+If a collaborator wants to upload data to a GCS bucket, you can follow this
+process to set up a service account that can be used to upload the data on their
+behalf:
+
+## Step 1: Create a service account
+
+- Log into your GCP account and select the project that contains the bucket in
+  which the data will be uploaded.
+- `IAM & Admin` → `Service Accounts` → `Create Service Account`.
+- Fill in the service account name - the ID should be filled in automatically.
+- (Optional) Add a description of the service account.
+- `Create`.
+
+<img src="figures/iam-create-service-account.png" alt="create service account" height="230" />
```

Comment:
Does this actually not look good using `![...](...)`? For my screenshots this seemed to have worked okay.

Reply:
Have you figured out a good way to resize figures with that syntax?
Most places I've looked suggest using raw HTML in the Markdown document.

Reply:
I don't think it's possible to resize with that syntax using GitHub's Markdown flavor. However, since it's screenshots, I usually didn't have to resize them and they still looked good.

Reply:
Okay, I'll use md syntax from now on. Just thought it's easier to adjust the actual figure size this way since screenshot sizes can vary.